internal/actions/pr.go

```diff
@@ -176,6 +176,7 @@ func CreatePullRequest(
 			Name:  defaultBranch,
 			Trunk: true,
 		}
+		branchMeta.Parent = parentState
```

Comment:
do you know why the `parentState` would be empty here to begin with? Just want to make sure we understand when does this particularly happen.

Reply:
The parent state is empty when the branch is not created with `av`, like `git checkout -b topic`.

Reply:
Can this also happen when an associated branch was deleted after merging a parent PR? I think that's what happened in my case.